react/src/components/UserDropdownMenu.tsx

```diff
@@ -55,6 +55,10 @@ const UserDropdownMenu: React.FC = () => {
       style: {
         color: '#1f1f1f',
         cursor: 'default',
+        maxWidth: '15vw',
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
```

Comment:
How about applying this using the `ellipsis` prop of `Typhora.Text`?

Reply:
If possible, setting style without explicit size and color is better for maintenance.